src/App.js

```diff
@@ -19,7 +20,8 @@ const rootReducer = combineReducers({
   activityReducer: activityReducer,
   landingReducer: landingReducer,
   profileReducer: profileReducer,
-  authReducer: authReducer
+  authReducer: authReducer,
+  autoLoginReducer: autoLoginReducer,
```

Comment:
Well I always think that if we create on separated file it will be easier (for coding, debugging, etc.). But yea, auto-login should just be in the /auth instead... I will fix it later, hopefully I will remember to do it.

Reply:
yea on a second thought (after reading your comment about the userData), I think these files should be left separately. Combining these reducers (which consequently combines their initialState) will be confusing, as their data (initial State) will be different from each other in term of structure and usage. Well, I don't even know what will you do with your userData, now and in the future.